Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/entity/Anchor.java

```diff
@@ -133,7 +132,7 @@ public void setThumbprint(String thumbprint) {
     public void setData(byte[] data) throws CertificateException {
         this.data = data;
 
-        if (this.data == Certificate.NULL_CERT) {
+        if (this.data == Certificate.getNullCert()) {
```

Comment:
Following-up on our offline discussion: this is code we inherited, and I believe that it is legitimately checking that "this.data" and "Certificate.getNullCert()" are the same object.  We should investigate as part of a Direct Config refactor later, but for now let's continue with logic that we know works.